src/main/webapp/index.html

```diff
@@ -16,6 +16,7 @@
         <div class="clear-icon">x</div>
       </div>
       <br>
+      <div id="modal" name="results-popup"></div>
```

Comment:
might be helpful to use 'class="modal"' if we're gonna be using modals for multiple displays, or if we're going to repopulate the same model with both search results and reviews, a different name may be more apt.

Reply:
I planned on only using one modal, disambiguation of name and id is imporant. 82e8aef